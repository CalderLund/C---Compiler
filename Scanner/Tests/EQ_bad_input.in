int wain(int a, int b) {
  if (a = = b) {
    a = 0;
  } else {}
  
  if (a == b) {
    a = 1;
  } else {}
  
  return a;
}
  
