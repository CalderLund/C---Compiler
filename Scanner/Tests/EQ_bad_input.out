INT int
WAIN wain
LPAREN (
INT int
ID a
COMMA ,
INT int
ID b
RPAREN )
LBRACE {
IF if
LPAREN (
ID a
BECOMES =
BECOMES =
ID b
RPAREN )
LBRACE {
ID a
BECOMES =
NUM 0
SEMI ;
RBRACE }
ELSE else
LBRACE {
RBRACE }
IF if
LPAREN (
ID a
EQ ==
ID b
RPAREN )
LBRACE {
ID a
BECOMES =
NUM 1
SEMI ;
RBRACE }
ELSE else
LBRACE {
RBRACE }
RETURN return
ID a
SEMI ;
RBRACE }
